packages/docs/src/routes/qwikcity/menu.md

```diff
@@ -58,7 +57,7 @@
 - [useDocumentHead()](api/use-document-head/index.mdx)
 - [useLocation()](api/use-location/index.mdx)
 
-## Middleware
+## Adaptors
```

Comment:
If you are changing this here, you probably want to update `middleware/overview/index.mdx` as well since the title is still using `Middleware`. Same thing with the pages under the `middleware` route.  I'm not sure how you would redirect all of the `/middleware/**` pages to `/adapters/**`, though. Possibly in the server side?